AutoDockTools/GridParameters.py

```diff
@@ -846,7 +847,8 @@ def getSideLengths(self, mol):
 
 
     def set_receptor(self, receptor_filename, gpf_filename=None):
-        self.receptor_filename = os.path.basename(receptor_filename)
+        # self.receptor_filename = os.path.basename(receptor_filename)
+        self.receptor_filename = receptor_filename
```

Comment:
What's wrong with `os.path.basename`?

Reply:
os.path.basename(receptor_filename) drops any information regarding file location when preparing the function. So, when you call the function with receptor_filename=/path/to/receptor.pdbqt with the receptor out of the working directory, the function didn't work as expected. 
It would be possible to store the receptor path in a separate variable, but I prefer to change the code as little as posible.